.github/workflows/push_pr.yaml

```
@@ -4,36 +4,30 @@ on:
     branches:
       - main
   pull_request:
+
 jobs:
   build:
-    name: Build integration for
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        goos: [ linux ]
-        goarch: [ amd64, arm64, arm ]
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-go@v2
-        with:
-          go-version: '1.17'
-      - uses: actions/cache@v2
-        with:
-          path: |
-            ~/.cache/go-build
-            ~/go/pkg/mod
-          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
-          restore-keys: |
-            ${{ runner.os }}-go-
-      - name: Build integration
-        env:
-          GOOS: ${{ matrix.goos }}
-          GOARCH: ${{ matrix.goarch }}
-        run: |
-          make compile
+    # Should be just:
+    # uses: ./.github/workflows/build.reusable.yml
+    # but https://github.com/github/feedback/discussions/10679
+    uses: newrelic/nri-kubernetes/.github/workflows/build.reusable.yml@ci/reusable-workflows
```

This is already solved. You can remove it if you wish.
```suggestion
    uses: ./.github/workflows/build.reusable.yml
```

---

It is not :(
https://github.com/newrelic/nri-kubernetes/actions/runs/1916515758

---

This is not a thing on the relative path but with the lack of `@main` which is even worse.